CLI: update GraalVM for native-images

GraalVM version used for generating native-images so far is now out-of-date (22.1.0), which is not supported any more. So, an update version is need.

Bellow is the error caught in the CLI Build workflow:

```
Error:  Failed to execute goal io.quarkus.platform:quarkus-maven-plugin:2.16.12.Final:build (default) on project dotcms-cli: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
Error: [ERROR] [error]: Build step io.quarkus.deployment.pkg.steps.NativeImageBuildStep#build threw an exception: 
        java.lang.IllegalStateException: Out of date version of GraalVM detected: GraalVM 22.1.0 Java 11 CE (Java Version 11.0.15+10-jvmci-22.1-b06). Quarkus currently supports 22.3.0. Please upgrade GraalVM to this version.
```

--- a/.github/workflows/cli-build-artifacts.yml
+++ b/.github/workflows/cli-build-artifacts.yml
@@ -19,7 +19,7 @@ on:
 env:
   JAVA_VERSION: 11
   JAVA_DISTRO: temurin
-  GRAALVM_VERSION: '22.1.0'
+  GRAALVM_VERSION: '22.3.1'
   BRANCH: ${{ inputs.branch || github.ref_name }}
   SKIP_TESTS: true
 

--- a/.github/workflows/cli-release-process.yml
+++ b/.github/workflows/cli-release-process.yml
@@ -21,7 +21,7 @@ defaults:
 env:
   JAVA_VERSION: 11
   JAVA_DISTRO: temurin
-  GRAALVM_VERSION: '22.1.0'
+  GRAALVM_VERSION: '22.3.1'
   MVN_PACKAGE_TYPE: 'uber-jar'
   NPM_PACKAGE_SCOPE: '@dotcms'
   NPM_PACKAGE_NAME: 'dotcli'

